charts/base/templates/pvc.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: {{ include "base.fullname" . }}
+  name: {{ .Values.storage.persistentVolumeClaimName }}
```

Comment:
make it optional to use .Values.storage.persistentVolumeClaimName so that by default it will get generated value from base.fullname and in case Values.storage.persistentVolumeClaimName passed it will use passed one